docs/book/src/design/asset-services.md

```diff
@@ -0,0 +1,15 @@
+# Asset services
+
+## Container diagram
+
+[![Container diagram: Asset services][diagram]][diagram]
+
+[diagram]: diagrams/CBDCServices-Container.svg
+
+**Figure:** [Container diagram] of the Nautilus asset services.
+
+[Container diagram]: https://c4model.com/#ContainerDiagram
+
+## Source code
+
+* <https://github.com/ntls-io/nautilus-wallet/tree/main/asset-services>
```

Comment:
For those less familiar with the ecosystem surrounding *digital assets*, I would suggest an  explanation or definition as well as some concrete example.  Similarly for the notion of an *asset service*.

Reply:
Yup, this still needs to have an overview filled in.

Reply:
Understood.  I was only trying to be thorough :nerd_face:

Reply:
This can be addressed in a follow-up PR and it's not like we'll forget about it (in this case) so this can be resolved